lib/saml/SamlIdpUrlBuilder.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+var njwt = require('njwt');
+var uuid = require('node-uuid');
+
+function SamlIdpUrlBuilder(application) {
+  this.application = application;
+  this.secret = application.dataStore.requestExecutor.options.client.apiKey.secret;
+}
+
+SamlIdpUrlBuilder.prototype._getServiceProvider = function _getServiceProvider(callback) {
+  this.application.getSamlPolicy(function (err, samlPolicy) {
+    if (err) {
+      return callback(err);
+    }
+
+    samlPolicy.getServiceProvider(callback);
+  });
+};
+
+SamlIdpUrlBuilder.prototype._buildInitializationUrl = function _buildInitializationUrl(initEndpointUrl, accessToken) {
+  return initEndpointUrl + '?accessToken=' + encodeURIComponent(accessToken);
```

Comment:
I just fixed this in my commit 431f79f57ae49879c927f3e16b59cbd4a7333e8a. I'm not totally sure about how the `accountStore` params are entered. Please review and come with suggestions.